src/main/java/webserver/request/Parser.java

```diff
@@ -38,6 +41,20 @@ public Request parse(BufferedReader in) throws IOException {
                 .build();
     }
 
+    public static HashMap<String, String> parseRequestBody(String body) {
```

Comment:
I think this is for JSON bodies, right? we should probably state that it's only for JSON bodies!

Reply:
Could we also add tests for this?